govc/tree/command.go

```diff
@@ -0,0 +1,183 @@
+/*
+Copyright (c) 2014-2015 VMware, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tree
+
+import (
+	"context"
+	"flag"
+	"os"
+	gopath "path"
+	"time"
+
+	gotree "github.com/a8m/tree"
+
+	"github.com/vmware/govmomi/find"
+	"github.com/vmware/govmomi/govc/cli"
+	"github.com/vmware/govmomi/govc/flags"
+	"github.com/vmware/govmomi/list"
+	"github.com/vmware/govmomi/vim25/types"
+)
+
+type tree struct {
+	*flags.DatacenterFlag
```

Comment:
Should use `ClientFlag` instead, since tree can work against multiple datacenters, don't want to require that `-dc` is set in that case.